### PR TITLE
Updating the list of custom formats with 'ktoml' project

### PR DESCRIPTION
**Pull request by akuleshov7**
Updating the format/Readme.md with 'Ktoml' project: multiplatform TOML parser and serializer for Kotlin Native.

### CODE REVIEW

**Comment by akuleshov7** [comment]
#1092

**Comment by akuleshov7** [comment]
@sandwwraith

File: formats/README.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ This library allows serialization and deserialization of objects to and from [Av
 
 Allows serialization and deserialization of objects to and from [BSON](https://docs.mongodb.com/manual/reference/bson-types/).
 
+### Ktoml 
+* GitHub repo: [akuleshov7/ktoml](https://github.com/akuleshov7/ktoml)
+* Artifact ID: `com.akuleshov7:ktoml-core`
+* Platforms: multiplatform, all Kotlin supported platforms
+
+Fully Native and Multiplatform Kotlin serialization library for serialization/deserialization of TOML format.
+This library contains no Java code and no Java dependencies. We believe that TOML is actually the most readable
```
Comment (sandwwraith) [inline formats/README.md:67]: Can you remove this sentence, please ('we believe that...')?. I don't argue that toml is good for configs, but this sentence is subjective, while this section should contain a plain description of what the library does.

**Review by sandwwraith — COMMENTED** [review]
Great, I think it'll be very useful

File: formats/README.md (hunk 1 above)
Comment (akuleshov7) [reply formats/README.md]: totally agree, @sandwwraith, updated

**Review by sandwwraith — APPROVED** [review]

**Pull request merged**